axelor-base/src/main/java/com/axelor/apps/base/service/PartnerService.java

```diff
@@ -103,6 +105,7 @@ public Partner createPartner(
   public void setPartnerFullName(Partner partner) {
 
     partner.setFullName(this.computeFullName(partner));
+    partner.setSimpleFullName(this.computeSimpleFullName(partner));
   }
 
   public String computeFullName(Partner partner) {
```

Comment:
Code of this method should be factorised as mentioned on ticket. Use computeSimpleFullName and add sequence to it in this method.